MatrixSDK/NotificationCenter/MXNotificationCenter.h

```diff
@@ -213,6 +229,19 @@ extern NSString *const kMXNotificationCenterAllOtherRoomMessagesRuleID;
                  sound:(BOOL)sound
              highlight:(BOOL)highlight;
 
+/**
+ Create a content push rule, see MXNotificationCenter notifications for operation result.
+ Uses pattern as ruleId.
+ @param pattern enable/disable notification.
+ @param notify enable/disable notification on events that match with the pattern.
+ @param sound enable/disable sound during notification.
+ @param highlight enable/disable highlight option.
+ */
+- (void)addContentRuleWithMatchingRuleIdAndPattern:(NSString *)pattern
```

Comment:
This may be easier to digest if it was `addContentRuleWithRuleIdFromPattern` or `addContentRuleWithRuleIdMatchingPattern`.